tests/test_cookiecutter_generation.py

```diff
@@ -93,6 +93,8 @@ def context():
     {"use_whitenoise": "n"},
     {"use_heroku": "y"},
     {"use_heroku": "n"},
+    {"use_watchman": "y"},
+    {"use_watchman": "n"},
```

Comment:
Watchman should not be used with `use_async="y"` option as uvicorn does not support watchman, only watchgod.